test/etl_test.py

```diff
@@ -131,3 +128,103 @@ def test_ferc1_solo_etl(datastore_fixture,
         pudl_settings_fixture,
         datapkg_bundle_name='ferc1-solo',
         clobber=True)
+
+
+class TestFerc1Datastore:
+    """Validate the Ferc1 Datastore and integration functions."""
+
+    def test_ferc_folder(self, pudl_ferc1datastore_fixture):
+        """Spot check we get correct folder names per dataset year."""
+        ds = pudl_ferc1datastore_fixture
+
+        assert ds.get_folder(1994) == "FORMSADMIN/FORM1/working"
+        assert ds.get_folder(2001) == "UPLOADERS/FORM1/working"
+        assert ds.get_folder(2002) == "FORMSADMIN/FORM1/working"
+        assert ds.get_folder(2010) == "UPLOADERS/FORM1/working"
+        assert ds.get_folder(2015) == "UPLOADERS/FORM1/working"
+
+    def test_get_fields(self, pudl_ferc1datastore_fixture):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=6263807)